src/OpaqueVal.cc

```diff
@@ -1073,7 +1073,7 @@ bool ParaglobVal::DoUnserialize(const broker::data& data)
 
 	try
 		{
-		this->internal_paraglob = build_unique<paraglob::Paraglob>(std::move(iv));
+		this->internal_paraglob = make_unique<paraglob::Paraglob>(std::move(iv));
```

Comment:
Isn't this missing `std::` as well as include for `<memory>`?

```suggestion
		this->internal_paraglob = std::make_unique<paraglob::Paraglob>(std::move(iv));
```

Reply:
Thanks, yeah, I'll fix in the merge commit.

There's some Zeek header files that we've been lazy to change for the longest time that are `using namespace std` -- I'd rather have full scoping in header, then up to source file to establish any namespace/scoping conveniences.  Might deserve elaborating/proposing what the convention should be in the style guide: https://docs.zeek.org/en/latest/devel/style_guide.html

(I'll make a PR/issue for that separate cleanup task)

Reply:
> I'd rather have full scoping in header, then up to source file to establish any namespace/scoping conveniences. Might deserve elaborating/proposing what the convention should be in the style guide.

I completely agree. `using namespace std` in header files is just asking for heartache anyways.